### PR TITLE
Fabiovelasquez

### PR DESCRIPTION
**Pull request by fvela009**
adding feature to display wishlist items

### CODE REVIEW

File: bookstore/settings.py [hunk 1]
```diff
@@ -90,6 +95,7 @@
         'ENGINE': 'django.db.backends.postgresql',
         'NAME': 'bookstoredb',
         'USER': 'postgres',
+        'PASSWORD': '123',
```
Comment (Mar-Sua) [inline bookstore/settings.py:98]: There are two passowrds here. If we merge, everyone will need to remember to delete one and update the pass.

**Review by Mar-Sua — APPROVED** [review]
Conflict files were resolved. Everything looks good to merge to main.

File: bookstore_api/urls.py [hunk 2]
```diff
@@ -1,10 +1,20 @@
 from django.urls import path
-from bookstore_api import views, users
- 
-urlpatterns = [ 
+from bookstore_api import views
+from .views import GenericAPIView
```
Comment (houyhourung) [inline bookstore_api/urls.py:3]: these imports and a few lines of codes need to be deleted. They give you errors and I remembered we already resolved it but not sure why the old codes still showed up #

**Review by houyhourung — CHANGES_REQUESTED** [review]
Please check bookstore_api/urls.py files. Those need to be the same as the main branch codes

File: bookstore_api/models.py [hunk 3]
```diff
@@ -25,5 +27,5 @@ class UserProfiles(models.Model):
     homeAddress = models.CharField(max_length=100)
     cart = models.JSONField(null=True)
     saveLater = models.JSONField(null=True)
-
+    wishlist = models.JSONField(null=True)
```
Comment (QuickWick123) [inline bookstore_api/models.py:30]: We have to wait until my merge request is approved and then we can add this 
from django.utils import timezone
and add
wishlist = models.JSONField(null=True)

File: bookstore_api/serializers.py [hunk 4]
```diff
@@ -11,4 +11,6 @@ class Meta:
 class UserSerializer(serializers.ModelSerializer):
     class Meta:
         model = UserProfiles
-        fields = '__all__'
\ No newline at end of file
+        fields = '__all__'
```
Comment (QuickWick123) [inline bookstore_api/serializers.py:14]: Same thing with this one. Serializers.py - nothing is being added to it and I have a total update on my merge. this file should not be updated

File: bookstore_api/urls.py (hunk 2 above)
Comment (QuickWick123) [reply bookstore_api/urls.py:3]: Agreed with houy, we should not delete ",users" and urlpatterns link must remain

**Pull request closed without merging**